### PR TITLE
tomee-webprofile 8.0.5

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
```
==> /usr/local/opt/tomee-webprofile/libexec/bin/configtest.sh
The operation couldn’t be completed. Unable to locate a Java Runtime.
Please visit http://www.java.com for information on installing Java.
```

File: Formula/tomee-webprofile.rb [hunk 1]
```diff
@@ -21,7 +23,8 @@ def install
     # Install files
     prefix.install %w[NOTICE LICENSE RELEASE-NOTES RUNNING.txt]
     libexec.install Dir["*"]
-    bin.install_symlink "#{libexec}/bin/startup.sh" => "tomee-webprofile-startup"
+    (bin/"tomee-webprofile-startup").write_env_script "#{libexec}/bin/startup.sh",
+        JAVA_HOME: Formula["openjdk"].opt_prefix
```
Comment (SMillerDev) [inline Formula/tomee-webprofile.rb:27]:
```suggestion
    (bin/"tomee-webprofile-startup").write_env_script "#{libexec}/bin/startup.sh",
        Language::Java.overridable_java_home_env
```

**Comment by SMillerDev** [comment]
The test and caveat should be fixed to account for the openjdk too

**Comment by chenrui333** [comment]
checkin

**Pull request closed without merging**